interfaces/apparmor/template.go

```diff
@@ -561,6 +561,7 @@ var defaultCoreRuntimeTemplateRules = `
   /{,usr/}bin/ldd ixr,
   /{usr/,}lib{,32,64}/ld{,32,64}-*.so ix,
   /{usr/,}lib/@{multiarch}/ld{,32,64}-*.so ix,
+  /{usr/,}lib/@{multiarch}/ld-linux-*.so* ix,
```

Comment:
There is much overlap with the rule above, the only reall difference is `so*` in this pattern. Snap confine uses: `/{,usr/}lib{,32,64,x32}/{,@{multiarch}/{,atomics/}}ld{-*,64}.so* mrix,`, so maybe the rule just above could be rewritten as: `/{usr/,}lib/@{multiarch}/ld{,32,64}-*.so* ix,`

Reply:
Very good point, I've changed to using the rule you suggest.